Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```
@@ -162,3 +162,46 @@
   - type: Item
     size: 5
     sprite: Objects/Weapons/Melee/e_cutlass.rsi
+
+- type: entity
+  name: Double Bladed Energy Sword
+  parent: EnergySword
+  id: EnergySwordDouble
+  description: Syndicate Command Interns thought that having one blade on the energy sword was not enough. This can be stored in pockets.
+  components:
+  - type: Wieldable
+    wieldTime: 1
+  - type: EnergySword
+    litDamageBonus:
+        types:
+            Slash: 6.5
+            Heat: 6.5
+            Structural: 6
+            Blunt: -4.5
+    litDisarmMalus: 0.6
```

```suggestion
    litDisarmMalus: 0.7
```

To make it worth 16 tc